src/components/AllArticles.jsx

```diff
@@ -0,0 +1,62 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import SingleArticle from './SingleArticle.jsx';
+
+//import { Route, Routes } from 'react-router-dom';
+
+function AllArticles() {
+  const [articles, setArticles] = useState([]);
+  const [isLoading,setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+  console.log('in All Articles');
+  useEffect(() => {
+   axios 
+     .get("https://be-my-news-example.herokuapp.com/api/articles")
+     .then(function (response) {
+       // handle success
+       console.log(response.statusText,'kkkkerror obj')
+       if (!response.statusText === 'OK') {
+        setIsError(true)
+        setIsLoading(false);
+       } else {
+        console.log(response.data.articles);
+        setIsLoading(false);
+        setArticles(response.data.articles);
+       }
+     });
+  }, []);
+  console.log('end of useEffect')
+  if (isLoading) return <p>Loading...</p>;
+  if (isError) return <p>Error happening........</p>
+
+  return (
+    <table>
```

Comment:
Please use other html tags to display your information. You will not be able to use <table>